<launch>
<node name="mavlink"
      pkg="rospilot"
      type="mavlink.py"
      respawn="false"
      output="screen"
      args="--baudrate=115200 --device=/dev/ardupilot --udp-export=127.0.0.1:8866" />
<node name="web_ui"
      pkg="rospilot"
      type="web_ui.py"
      respawn="false"
      output="screen"
      args="--media_path=~/.rospilot/media" />
<node name="vlc"
      pkg="rospilot"
      type="ros_vlc.py"
      respawn="false"
      output="screen"
      args="--media_path=~/.rospilot/media" />
<node name="camera"
      pkg="rospilot_deps"
      type="raw_usb_cam_node"
      respawn="false"
      output="screen">
    <param name="image_width" value="960" />
    <param name="image_height" value="720" />
    <param name="video_device" value="/dev/video1" />
</node>
<node name="mjpeg"
      pkg="rospilot_deps"
      type="raw_mjpeg_server"
      respawn="false"
      output="screen" />
<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
     <arg name="port" value="8088"/>
</include>
</launch>
